File: app/src/main/java/in/ac/iiita/go/models/LectureFragment.java
package in.ac.iiita.go.models;


import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import in.ac.iiita.go.R;

/**
 * A simple {@link Fragment} subclass.
 */
public class LectureFragment extends Fragment {
    
    
    public LectureFragment() {
        
    }
    
    
    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_lecture, container, false);
    }
    
}

File: app/src/main/java/in/ac/iiita/go/fragments/LectureFragment.kt
package `in`.ac.iiita.go.fragments


import `in`.ac.iiita.go.R
import `in`.ac.iiita.go.adapter.LectureViewPagerAdapter
import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v7.app.AppCompatActivity
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import kotlinx.android.synthetic.main.fragment_lecture.*
import org.jetbrains.anko.AnkoLogger
import org.jetbrains.anko.info

class LectureFragment : Fragment(), AnkoLogger {

    override fun onCreateView(inflater: LayoutInflater?, container: ViewGroup?,
                              savedInstanceState: Bundle?): View? {

        info("Here")
        return inflater!!.inflate(R.layout.fragment_lecture, container, false)
    }

    override fun onViewCreated(view: View?, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)
        viewPager.adapter = LectureViewPagerAdapter(activity.supportFragmentManager)
        tabLayout.setupWithViewPager(viewPager)

        (activity as AppCompatActivity).supportActionBar!!.title = "Lecture Schedule"
    }
}
